pkg/version/version.go

```diff
@@ -0,0 +1,40 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package version
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/submariner-io/admiral/pkg/log"
+)
+
+var (
+	version         = "devel"
+	goRuntimeString = fmt.Sprintf("%s / %s / %s", runtime.Compiler, runtime.Version(), runtime.GOARCH)
+	gitCommit       string
+	gitBuildDate    string
+)
+
+func PrintVersion(logger *log.Logger) {
+	logger.Info(fmt.Sprintf("Go Version: %s", goRuntimeString))
+	logger.Info(fmt.Sprintf("Git Commit: %s", gitCommit))
+	logger.Info(fmt.Sprintf("Git Commit Time: %s", gitBuildDate))
+	logger.Info(fmt.Sprintf("Submariner version: %v", version))
```

Comment:
"Submariner version" is not consistent with "Go Version", and different formats are used ("%s" vs "%v").